featuretools/primitives/primitive_base.py

```diff
@@ -100,12 +102,14 @@ def entity(self):
     # P TODO: this should get refactored to return_type
     @property
     def variable_type(self):
+        from . import direct_feature
```

Comment:
Should we move this import up to the top?

Reply:
direct_feature imports this file, so we have to include this here to avoid the circular reference. 

in #326, the directory structure changes things around and actually moves `DirectFeature` into this file, so we'll likely be able to update this when that PR gets merged in.

Reply:
makes sense